rocketpy/motors/SolidMotor.py

```diff
@@ -1,754 +1,770 @@
+        Motor.grainDensity : float
+            Density of each grain in kg/meters cubed.
+        Motor.grainOuterRadius : float
+            Outer radius of each grain in meters.
+        Motor.grainInitialInnerRadius : float
+            Initial inner radius of each grain in meters.
+        Motor.grainInitialHeight : float
+            Initial height of each grain in meters.
+        Motor.grainInitialVolume : float
+            Initial volume of each grain in meters cubed.
+        Motor.grainInnerRadius : Function
+            Inner radius of each grain in meters as a function of time.
+        Motor.grainHeight : Function
+            Height of each grain in meters as a function of time.
+
+        Mass and moment of inertia attributes:
+        Motor.centerOfMass : Function
+            Position of the center of mass in meters as a function of time.
+            Constant for solid motors, as the grains are assumed to be fixed.
+            See `Motor.coordinateSystemOrientation` for more information
+            regarding the motor's coordinate system
+        Motor.grainInitialMass : float
+            Initial mass of each grain in kg.
+        Motor.propellantInitialMass : float
+            Total propellant initial mass in kg.
+        Motor.mass : Function
+            Propellant total mass in kg as a function of time.
+        Motor.massDot : Function
+            Time derivative of propellant total mass in kg/s as a function
+            of time.
+        Motor.inertiaI : Function
+            Propellant moment of inertia in kg*meter^2 with respect to axis
+            perpendicular to axis of cylindrical symmetry of each grain,
+            given as a function of time.
+        Motor.inertiaIDot : Function
+            Time derivative of inertiaI given in kg*meter^2/s as a function
+            of time.
+        Motor.inertiaZ : Function
+            Propellant moment of inertia in kg*meter^2 with respect to axis of
+            cylindrical symmetry of each grain, given as a function of time.
+        Motor.inertiaDot : Function
+            Time derivative of inertiaZ given in kg*meter^2/s as a function
+            of time.
+
+        Thrust and burn attributes:
+        Motor.thrust : Function
+            Motor thrust force, in Newtons, as a function of time.
+        Motor.totalImpulse : float
+            Total impulse of the thrust curve in N*s.
+        Motor.maxThrust : float
+            Maximum thrust value of the given thrust curve, in N.
+        Motor.maxThrustTime : float
+            Time, in seconds, in which the maximum thrust value is achieved.
+        Motor.averageThrust : float
+            Average thrust of the motor, given in N.
+        Motor.burnOutTime : float
+            Total motor burn out time, in seconds. Must include delay time
+            when the motor takes time to ignite. Also seen as time to end thrust
+            curve.
+        Motor.exhaustVelocity : float

```

Comment:
Need to add `burn_time` here

Reply:
Solved by [081f0de](https://github.com/RocketPy-Team/RocketPy/pull/337/commits/081f0de83c750b1e936a149b49759e5f97e8ea96).